resotolib/resotolib/graph/__init__.py

```diff
@@ -402,6 +402,22 @@ def resource_classes_to_resotocore_model(classes: Set[Type[Any]], **kwargs: Any)
     return list(model.values())
 
 
+def export_model() -> List[Json]:
+    """Return the graph node dataclass model in resotocore format"""
+    model = dataclasses_to_resotocore_model({BaseResource}, aggregate_root=BaseResource)
```

Comment:
```suggestion
    model = resource_classes_to_resotocore_model({BaseResource}, aggregate_root=BaseResource)
```

Or maybe the 2 methods should be combined into one.

Reply:
Good idea, done.